views/pages/players.html

```diff
@@ -12,4 +12,95 @@
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="club" class="block text-sm font-medium leading-6 text-gray-900">Club:</label>
+          <div class="text-sm">
+            <input type="text" id="club" name="club" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="position" class="block text-sm font-medium leading-6 text-gray-900">Position:</label>
+          <div class="text-sm">
+            <select id="position" name="position" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+                <option value="goalkeeper">Goalkeeper</option>
+                <option value="defender">Defender</option>
+                <option value="midfielder">Midfielder</option>
+                <option value="forward">Forward</option>
+            </select>
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="manager_name" class="block text-sm font-medium leading-6 text-gray-900">Manager Name:</label>
+          <div class="text-sm">
+            <input type="text" id="manager_name" name="manager_name" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="manager_name" class="block text-sm font-medium leading-6 text-gray-900">Manager Name:</label>
+          <div class="text-sm">
+            <input type="text" id="manager_name" name="manager_name" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="telephone" class="block text-sm font-medium leading-6 text-gray-900">Telephone:</label>
+          <div class="text-sm">
+            <input type="tel" id="telephone" name="telephone" pattern="[0-9]{10}" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
```

Comment:
I'm not convinced that it's worth sanitizing telephone numbers with `[0-9]{10}`. That would have issues with country codes, and many commonly used formats like `+1 (555) 555-5555`

We probably won't be doing anything with this field except displaying it, so as long as we have some sort of length limit in the backend (which we'd want for many fields), I'd recommend leaving it as an unsanitized string.